### DISCUSSION TITLE
Is OpenModelica 1.24.0~dev-16 already stable?

### DISCUSSION THREAD

**Discussion by pierre-haessig** [OpenModelica/OpenModelica#12636] [Q&A] [answered]
Hello,
I'm using OpenModelica on Debian bookworm from the "stable" channel. The website's [Linux install page](https://openmodelica.org/download/download-linux/) tells that, as of today (2024-04-24) the latest stable release is 1.23.0\~3-g4080a7b. However, I see that I received (when?) 1.24.0\~dev-16-gaa27cc1-1.

Is this 1.24 dev version already considered stable enough (and then the website needs an update)?

**Comment by AnHeuermann** [comment]
The nightly version will be update every night with the latest developments form the master branch. We test a lot but from time to time some regressions find their way on the master. So the nightly version isn't stable and can contain bugs.

**Reply by pierre-haessig** [reply]
Fine, but then why is it in the *stable* channel for Linux distributions?

**Reply by casella** [reply]
@pierre-haessig in the past we had Linux users that needed some features for activities like teaching a course, but those features were still not available in an official release. To support this case, we made a stable release when were were decently sure that the daily commits hadn't broken anything. We haven't use this feature for a few years, but it's always available, just in case. 

Stable also hosts the beta releases as soon as they become available.

**Reply by pierre-haessig** [reply, marked as answer]
OK sorry, it was probably a misconfiguration on my side: I can see in https://build.openmodelica.org/apt/dists/bookworm/stable/binary-amd64/Packages that indeed the `stable` channel is serving version `1.23.0~3-g...`, not a nightly 1.24 build:

```
Package: drcontrol
Architecture: all
Version: 1.23.0~3-g4080a7b-1
...
```

I don't know from where I installed that version `1.24.0~dev-16-gaa27cc1-1` (which is not even `nightly` since this channel currently serves version [1.24.0~dev-165](https://build.openmodelica.org/apt/dists/bookworm/nightly/binary-amd64/Packages)), since it my APT source configuration file  `openmodelica.list`, I only have the `stable` channel:
```
deb http://build.openmodelica.org/apt bookworm stable
```

However, perhaps some months ago, I had made a temporary change?

Anyway, after removing version `1.24.0~dev-16...`, only version `1.23.0~3...` was listed by `aptitude` as available and it installed just fine. Sorry for the noise.

**Reply by casella** [reply]
No problem
